ZCRMiOS/Classes/ZCRMModuleRelation.swift

```diff
@@ -298,7 +305,7 @@ public class ZCRMModuleRelation : ZCRMEntity
     /// - Throws: ZCRMSDKError if failed to get notes of the ZCRMRecord
     public func getNotes(ofParentRecord : ZCRMRecord, page : Int, per_page : Int, sortByField : String?, sortOrder : SortOrder?, modifiedSince : String?, completion : @escaping( BulkAPIResponse?, Error? ) -> () )
 	{
-        self.getNotes(ofParentRecord: ofParentRecord, page: 1, per_page: 20, sortByField: nil, sortOrder: nil, modifiedSince: nil) { ( response, error ) in
+        RelatedListAPIHandler(parentRecord: ofParentRecord, relatedList: self).getNotes(ofParentRecord: ofParentRecord, page: page, per_page: per_page, sortByField: sortByField, sortOrder: sortOrder, modifiedSince: modifiedSince) { ( response, error ) in
```

Comment:
Why does RelatedListAPIHandler.getNotes need the parentRecord argument when parentRecord is a property in RelatedListAPIHandler class? getNotes can have similar method signature like getRecords.